intrinsic_sdk_cmake/cmake/fetch_sdk.cmake

```diff
@@ -4,8 +4,8 @@ include(FetchContent)
 # Fetch the intrinsic sdk source code during configure stage.
 FetchContent_Declare(
   intrinsic_sdk
-  URL https://github.com/intrinsic-ai/sdk/archive/refs/tags/v1.16.20250210.tar.gz
-  URL_HASH SHA256=be55bdb2c761b7d55476c2ee843c4e444e0953f03aafce2481797eec98e19ac0
+  URL https://github.com/intrinsic-ai/sdk/archive/refs/tags/v1.16.20250303.tar.gz
+  URL_HASH SHA256=2181ae09684bc35e47b251e21d969a4e0f3cbf72818470caaa416e2834748d32
```

Comment:
I'd recommend building off the other two open prs. Now that it's been tested I should merge those, but they already update the sdk.